src/test/kotlin/se/uulm/snowballr/backend/service/user/UpdateUserTest.kt

```diff
@@ -104,7 +121,48 @@ class UpdateUserTest : MainServiceTest() {
     }
 
     @Test
-    fun `When update succeeds, then updated user is returned`() = runTest {
+    fun `When user updates own email, then it succeeds`() = runTest {
+        val currentUser = DataBuilder.createExampleUser(id = requestedUserId, role = UserRole.USER_ROLE_DEFAULT)
+        val request = UserOuterClass.User.Update.newBuilder()
+            .setUser(
+                UserOuterClass.User.newBuilder()
+                    .setId(currentUser.id.toString())
+                    .setEmail("new-and-shiny@email.com"),
```

Comment:
I think we should stick with `@example.com`, as that domain is dedicated to illustrative examples. Some actually owns `@email.com`